boltdesignsystem/bolt
# DS-195: Page Level Information Density Feature

## Jira

https://pegadigitalit.atlassian.net/browse/DS-195

## Summary

Adds the information density feature by using `data-bolt-info-density`.

## Details

1. `data-bolt-info-density` is created, it accepts compact and comfortable.
1. Compact will condense the spacing and typography.
1. Comfortable will expand the spacing and typography.
1. If `data-bolt-info-density` is not set, it uses the regular system.

## How to test

Run the branch locally and check the Information Density docs under Visual Styles.
